num_to_words/__init__.py

```diff
@@ -29,9 +31,7 @@ def num_to_word(num, lang, separator=", ", combiner=" "):
     for language in SUPPORTED_LANGUAGES:
         for num_index in range(10):
             num = num.replace(DIGITS_LANG_MAPPING[language][num_index], DIGITS_LANG_MAPPING["en"][num_index])
-
-    # Assert that input contains only integer number
-    assert num, "Input string is empty"
+    num = language_specific_exception(num, lang, combiner)
```

Comment:
this function has been planned to handle exception after the conversation,
using this function does not feel right, we can have two function for the exception,
- one before the converstion (what you are trying to do)
- one after the conversation (what the function `language_specific_exception` is currently doing)